WW3_PreProc/namelists_Global.nml

```diff
@@ -1,9 +1,11 @@
 &SIN6 SINA0=0.04 /
 &SWL6 SWLB1=0.22E-03, CSTB1=T /
 &SNL1 LAMBDA=0.237, NLPROP=2.13E+07 /
-&OUTS E3D = 1, USSP = 1, IUSSP=3, STK_WN = 0.04, 0.11, 0.33 /
-&CALTYPE = '360_day' /
+&CALTYPE = '365_day' /
+&OUTS E3D = 1 /
 &LMPN LMPENABLED = T, HSLMODE = 1 /
 &MISC CICE0 = 0.25, CICEN = 0.75, FLAGTR = 4 /
-&PRO3 WDTHCG = 2.00, WDTHTH = 2.00 /
+&SIC3 IC3CHENG=.TRUE., USECGICE=.FALSE.,  
+  IC3VISC = 1.0e3, IC3DENS = 917.0, IC3ELAS = 1.0e3 /
+&IS2ANDISB=.TRUE., IS2BACKSCAT =0.2  /
```

Comment:
I haven't checked these. Could you please double check that the changes are correct.

Reply:
These changes are correct